src/components/WhimsicalInstaller/WhimsicalInstaller.helpers.js

```diff
@@ -15,6 +16,19 @@ export type FileStatus =
   | 'caught' // The user is grabbing the file
   | 'released'; // The user has released a previously-grabbed file
 
+export type FileData = {
+  id: string,
+  x: number,
+  y: number,
+  size: number,
+  status: FileStatus,
+  flightPath: BezierPath,
+  speed?: {
+    horizontalSpeed: number,
+    verticalSpeed: number,
+  },
```

Comment:
This API feels a bit weird to me, since this `speed` property only exists for released files :/ suggestions welcome